README.md

```diff
@@ -158,19 +159,17 @@ models:
       owner: "@meder"
       fal:
       	scripts:
-          before:
-            - fal_scripts/postgres.py
-  	      after:
+          pre-hook:
+            - fal_scripts/trigger_fivetran.py
+  	      post-hook:
             - fal_scripts/slack.py
 ```
 
-`fal run --before` will run `fal_scripts/postgres.py` script regardless if dbt has calculated the boston model or not. `fal run` without the `--before` flag, will run `fal_scripts/slack.py`, but only if boston model is already calculated by dbt.
+`fal flow run` will run `fal_scripts/trigger_fivetran.py`, then the `boston` dbt model, and finally `fal_scripts/slack.py`. If you select a model to be run with a selection flag (e.g. `--select boston`, the hooks associated to the model will always run with it.
```

Comment:
If a model is selected specifically with a selection flag (e.g. `--select boston`), the hooks associated to the model will always run with it.